docs/reference/settings/ilm-settings.asciidoc

```diff
@@ -59,6 +59,15 @@ An index that was rolled over would normally match the full format,
 for example `logs-2016.10.31-000002`). 
 If the index name doesn't match the pattern, index creation fails.
 
+[[index-lifecycle-step-wait-time-threshold]]
+`index.lifecycle.step.wait_time_threshold`::
+(<<indices-update-settings,Dynamic>>, <<time-units,time value>>) 
+Time to wait while retrying a failed allocation step for the {ilm-init}
+<<ilm-shrink,`shrink`>> action. After this period, {ilm-init} re-attempts the
```

Comment:
@jrodewig "re-attempts the previous step" is not entirely accurate as the shrink action rewinds back a few steps, depending on the situation. Maybe we shouldn't be so prescriptive here and say the `shrink` action (which is the only place where this is used now) "does the right thing"

Reply:
This is a good point. We don't really need to document the remedial action here anyway. That's already covered in the xref.